### PR TITLE
Add bob.db.avspoof recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

**Comment by 183amir** [comment]
needs a restart on the ci.

**Comment by jakirkham** [comment]
Restarted.

File: recipes/bob.db.avspoof/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - six
+    - sqlalchemy
+    - bob.io.base
+    - bob.db.base
+    - antispoofing.utils
+    - bob.db.verification.utils
+
+test:
+  commands:
+    - bob_dbmanage.py avspoof download
```
Comment (jakirkham) [inline recipes/bob.db.avspoof/meta.yaml]: What is this doing just out of curiosity?

Comment (183amir) [inline recipes/bob.db.avspoof/meta.yaml]: downloads the file list of the database which is in a sql database. The database is required for nosetests.

**Comment by msarahan** [comment]
Nice, clean recipe.  Thanks @183amir

**Comment by msarahan** [comment]
This one failed to create a feedstock because we got github rate limited.  @pelson @ocefpaf @jakirkham  what repair actions are necessary?

**Comment by jakirkham** [comment]
Wait 20mins and restart Travis CI. That's what I have been doing. Maybe we can wait less than that, but I don't want to get rate limited again and have to wait longer.

**Comment by jakirkham** [comment]
Normally I have also tried to open an issue so people know about this, but it hasn't been that effective about making people aware as I would like. Here's the current one ( https://github.com/conda-forge/staged-recipes/issues/529 ). Was thinking about a status page ( https://github.com/conda-forge/conda-forge.github.io/issues/78 ) to make this stuff more visible so we can see it, track it, maybe get notified, collect stats, etc.

**Pull request merged**